.buildkite/scripts/generate_xpack_osquerybeat_pipeline.sh

```diff
@@ -181,7 +181,10 @@ if are_conditions_met_packaging; then
     steps:
       - label: ":linux: Packaging Linux"
         key: "packaging-linux"
-        command: "cd $BEATS_PROJECT_NAME && mage package"
+        command: |
+          sudo apt-get update -y
+          DEBIAN_FRONTEND=noninteractive sudo apt-get install --no-install-recommends --yes msitools
```

Comment:
what's the reason for not using the existing script?

```suggestion
          .ci/scripts/install-msitools.sh
```

Reply:
We are trying not to use the `.ci` scripts in buildkite due to migration. Because it is going to be removed by EoM.

Reply:
what's the reason you don't use `.buildkite/scripts` and then apply the changes in the `Jenkinsfile`?

Reply:
If I understand the question correctly, the `.buildkite/scripts` changes are related to the osquerybeat packaging in bk

Reply:
sorry if I was vague, I meant what's the reason you don't rename `.ci/scripts/install-msitools.sh` for `.buildkite scripts/install-msitools.sh` and then use it in both places (Jenkins and Buildkite pipelines).

IIUC, `.ci` will be removed but `.buildkite` will remain, therefore you can take advantage of removing the duplicated code for `apt-get update` and `apt-get install msitools` using the `.buildkite/scripts/install-msitools.sh` and use that script in both CI pipelines.

Reply:
Ok. I updated the branch and now this script is deleted

Reply:
We will add the `msitools` to the base image and delete all install scripts after that

Reply:
Raised a [follow up issue](https://github.com/elastic/ingest-dev/issues/3194) for that